.github/workflows/conventional_commits.yml

```diff
@@ -0,0 +1,41 @@
+name: Conventional Commits
+
+on:
+  pull_request_target:
+    types:
+    - opened
+    - edited
+    - synchronize
+    - reopened
+
+permissions:
+  pull-requests: read
+
+jobs:
+  commit-lint:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    - name: Lint Commits
+      uses: wagoid/commitlint-github-action@v6
+    - name: Lint Pull Request
+      uses: amannn/action-semantic-pull-request@v5.5.3
+      env:
+        GITHUB_TOKEN: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
```

Comment:
`CUSTOM_GITHUB_TOKEN` has more permissions than the built-in one. Can we just use the built-in token (each action will automatically create a token for the action and delete it after the action is finished) instead?

ref: https://github.com/wagoid/commitlint-github-action#token